### DISCUSSION TITLE
560,000 Database not returning some values in querys

### DISCUSSION THREAD

**Discussion by EliasHodkinson** [supabase/supabase#29668] [Questions]
Hey!

I am trying to query my database that has 570,000 rows. It appears that whenever I make a call to the database for any rows between 60,000 to 560,000 I will get a return of []. Was just wondering if there is a limit or timeout happening when trying to search a record in a database this big?

When using the API's to make this type of call, do I need to add some other type of parameter to allow the request to wait longer for a response?

I am completely new to databases so its probably me. But was hoping for some guidance if possible. I am writing a macOS Swift app.

**Comment by GaryAustin1** [comment]
You should show your call and if using the REST API data and error.  Normally if you get empty array then you have no error and no timeout.

If you have an error you could also check the dashboard postgres logs for more detail.

**Comment by EliasHodkinson** [comment]
Apologies Gary. Upon doing some BASIC troubleshooting steps, turns out I didnt upload my entire database correctly, as such it was not finding the record lol. Thanks for your assistance, sorry to waste your time!
